### DISCUSSION TITLE
Is there any other open source design system that uses React Aria and React Stately other than React Spectrum?

### DISCUSSION THREAD

**Discussion by oggnimodd** [adobe/react-spectrum#3874] [General]
Just curious how people use React Aria and React Stately to build their own design systems.

**Comment by devongovett** [comment]
There are probably a lot more of you search GitHub, but here's a few I found a while ago:

- https://github.com/prisma/lens
- https://github.com/homebound-team/beam
- https://github.com/envoy/polarwind

**Reply by oggnimodd** [reply]
Nice, I found these on GitHub :

- https://github.com/cube-js/cube-ui-kit
- https://github.com/marigold-ui/marigold
